src/m365/spo/commands/page/page-control-set.ts

```diff
@@ -120,7 +120,43 @@ class SpoPageControlSetCommand extends SpoCommand {
           }
         }
 
-        return Page.save(pageName, args.options.webUrl, canvasContent, logger, this.debug, this.verbose);
+        let pageData: any = {};
```

Comment:
Aren't we duplicating the logic that we'll need in other commands as well? Shouldn't this be centralized in `pageMethods.ts`?

Reply:
Yes, that is what I'm planning to do, but requires more time. 

Once these fixes are in, I want to take a look at all page command and create a new base class for all of them.

Reply:
Got it. Shall we create an issue for it not to lose track of it?

Reply:
Sure